src/components/general/Blog.vue

```diff
@@ -9,12 +25,122 @@ export default {
 	props: {
 		limit: Number
 	},
+	data() {
+		return {
+			blogData: []
+		}
+	},
+	methods: {
+		toText(htmlText) {
```

Comment:
This method essentially takes the description that was returned from PHP (which has HTML elements in it) and converts it to plain text to avoid the HTML elements.